ads-txt.php

```diff
@@ -49,6 +50,25 @@ function tenup_display_ads_txt() {
 			echo esc_html( apply_filters( 'ads_txt_content', $adstxt ) );
 			die();
 		}
+	} elseif ( '/app-ads.txt' === $request ) {
+		$post_id = get_option( APP_ADS_TXT_MANAGER_POST_OPTION );
+
+		// Will fall through if no option found, likely to a 404.
+		if ( ! empty( $post_id ) ) {
+			$post = get_post( $post_id );
```

Comment:
Likelihood of an issue here is pretty low (or probably even zero) but we might want to check that this `get_post` call returns an actual post object prior to using it. If by some chance a wrong ID gets sent here, this would throw an error.